File: java/src/test/java/mx/iteso/adapter/MSLicenseTest.java
package mx.iteso.adapter;

import static org.junit.Before;
import static org.junit.Test;
import static org.junit.Assert.assertEquals;


import mx.iteso.adapter.impl.other.MSLicense;

/**
 * MSLicense test class.
 */
public class MSLicenseTest {
    /**
     * Validation boolean.
     */
    private MSLicense license;
    /**
     * Set up method.
     */
    @Before
    public void SetUp() {
        license = new MSLicense();
    }
    /**
     * Validation test.
     */
    @Test
    public void validationTest() {
        assertEquals(true, license.isValid());
    }

}

File: csharp/adapter_practice/test/MSLicenseTest.cs
using adapter_practice.main.other;
using NUnit.Framework;
using System;
using System.Collections.Generic;
using System.Text;

namespace adapter_practice.test
{
    [TestFixture]
    class MSLicenseTest
    {
        private MSLicense license;
        /**
         * Set up method.
         */
        [SetUp]
        public void SetUp()
        {
            license = new MSLicense();
        }
        /**
         * Validation test.
         */
        [TestCase]
        public void validationTest()
        {
            Assert.AreEqual(true, license.isValid());
        }
    }
}
}
